src/components/pokedex.css

```diff
@@ -181,41 +182,42 @@
 .heart-button {
   position: absolute;
```

Comment:
ok this is super cracy 😅  Try to avoid this! 

instead you'd need to use flexbox for positioning these elements and the panels
 
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

xD